CONTRIBUTING.md

```diff
@@ -1,48 +1,67 @@
-# Contributing to ZenML
+# 🧑‍💻 Contributing to ZenML
 
-A big welcome and thank you for considering contributing to ZenML! It’s people like you that make it a reality for users in our community.
+A big welcome and thank you for considering contributing to ZenML! It’s people like you that make it a reality for users
+in our community.
 
-Reading and following these guidelines will help us make the contribution process easy and effective for everyone involved. It also communicates that you agree to respect the time of the developers managing and developing these open source projects. In return, we will reciprocate that respect by addressing your issue, assessing changes, and helping you finalize your pull requests.
+Reading and following these guidelines will help us make the contribution process easy and effective for everyone
+involved. It also communicates that you agree to respect the time of the developers managing and developing these open
+source projects. In return, we will reciprocate that respect by addressing your issue, assessing changes, and helping
+you finalize your pull requests.
 
-## Quicklinks
+## ⚡️ Quicklinks
 
 * [Code of Conduct](#code-of-conduct)
 * [Getting Started](#getting-started)
     * [Issues](#issues)
     * [Pull Requests](#pull-requests)
 * [Getting Help](#getting-help)
 
-## Code of Conduct
+## 🧑‍⚖️ Code of Conduct
 
-We take our open source community seriously and hold ourselves and other contributors to high standards of communication. By participating and contributing to this project, you agree to uphold our [Code of Conduct](https://github.com/zenml-io/zenml/blob/master/CODE-OF-CONDUCT.md).
+We take our open source community seriously and hold ourselves and other contributors to high standards of
+communication. By participating and contributing to this project, you agree to uphold
+our [Code of Conduct](https://github.com/zenml-io/zenml/blob/master/CODE-OF-CONDUCT.md).
 
-## Getting Started
+## 🛫 Getting Started
 
 Contributions are made to this repo via Issues and Pull Requests (PRs). A few general guidelines that cover both:
 
-- To report security vulnerabilities, please contact [support@zenml.io](mailto:support@zenml.io), which is monitored by our security team.
+- To report security vulnerabilities, please contact [support@zenml.io](mailto:support@zenml.io), which is monitored by
+  our security team.
 - Search for existing Issues and PRs before creating your own.
-- We work hard to makes sure issues are handled in a timely manner but, depending on the impact, it could take a while to investigate the root cause. A friendly ping in the comment thread to the submitter or a contributor can help draw attention if your issue is blocking.
+- We work hard to makes sure issues are handled in a timely manner but, depending on the impact, it could take a while
```

Comment:
```suggestion
- We work hard to make sure issues are handled on time, but it could take a while to investigate the root cause depending on the impact.

```